networking/src/p2p/stream.rs

```diff
@@ -269,6 +280,21 @@ impl EncryptedMessageReader {
         loop {
             // read
             let message_encrypted = self.rx.read_message().await?;
+            let message_len = message_encrypted.len();
+
+            // check that if the chunk is not a first chunk of the message (input_remaining > 0)
+            // and not a last chunk of the message (message_len > input_remaining + BOX_ZERO_BYTES),
+            // then it should not be shorter than `CONTENT_LENGTH_MIN`
+            if input_remaining > 0
+                && message_len > input_remaining + BOX_ZERO_BYTES
+                && message_len < CONTENT_LENGTH_MIN
+            {
+                debug!(self.log, "Chunk is too short"; "remaining" => input_remaining);
```

Comment:
@akoptelov 
do we need this debug, even if we throw error?
I would better add just more info (input_remaining, ...) to StreamError::ChunkSizeError,
and if the error is propagated, it will be logged by caller

Reply:
It was rather for my debugging purposes, I think we can remove it.